sympy/functions/elementary/trigonometric.py

```diff
@@ -20,6 +20,13 @@
 from sympy.polys.specialpolys import symmetric_poly
 from sympy.utilities.iterables import numbered_symbols
 
+def _imaginary_unit_as_coefficient(arg):
```

Comment:
This function seems out of place at this line in the file.

Reply:
@oscarbenjamin I moved it to `sympy.core.expr`.